.github/workflows/check.yml

```diff
@@ -36,11 +36,11 @@ jobs:
           - "2.7"
           - pypy-2.7
         os:
-          - ubuntu-20.04
-          - macos-10.15
-          - windows-2022
```

Comment:
I like the explicit nature here of the version used then this link. If some explicit version needs bumped we can do that separately.